Refine Go: Libraries (#3317)
open-telemetry/opentelemetry.io: content/en/docs/instrumentation/go/libraries.md

## Setup

Each instrumentation library is a package. In general, this means you need to

> `go get` the appropriate package. For example, for the instrumentation libraries
> maintained in the
> [Contrib repository](https://github.com/open-telemetry/opentelemetry-go-contrib)
> this means:

Reviewer: nit: try to avoid repeated use of "for"

```suggestion
`go get` the appropriate package. For example, to get the instrumentation libraries
maintained in the
[Contrib repository](https://github.com/open-telemetry/opentelemetry-go-contrib)
run the following:
```

```sh
go get go.opentelemetry.io/contrib/instrumentation/{import-path}/otel{package-name}
```

Then configure it in your code based on what the library requires to be
activated.

> [Getting Started](../manual/) provides an example, how you can set up
> instrumentation for a `net/http` server.

Reviewer: ```suggestion
[Getting Started](../manual/) provides an example showing how to set up
instrumentation for a `net/http` server.
```
